src/main/java/uk/gov/hmcts/reform/idam/health/info/ForgerockVersionInfoContributor.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.reform.idam.health.info;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.collect.ImmutableMap;
+import lombok.CustomLog;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.actuate.info.Info;
+import org.springframework.boot.actuate.info.InfoContributor;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.format.DateTimeFormatter;
+import java.util.stream.Collectors;
+
+@CustomLog
+@Component
```

Comment:
I guess only nitpick would be, its not immediately obvious that this is the git commit of the packer branch.
This is the one time I will say we need a comment on the class